### DISCUSSION TITLE
Top Solid Infill

### DISCUSSION THREAD

**Discussion by Bfinny** [supermerill/SuperSlicer#1908] [Q&A] [answered]
I'm wondering how to turn the "top Solid Infill" off (shown in red)

![image](https://user-images.githubusercontent.com/90668637/142253106-ec85f4f9-44e7-4423-805e-41912aec24fa.png)

**Comment by supermerill** [comment, marked as answer]
you have to:
 * select the same pattern as the solid infill
 * put the same top speed (or set 100%) as the solid fill
 * uncheck "only one peri on top" (as you don't want a top)
 
And ignore the color, as it's now the same thing.

Note that this is a calibration. They have object modifiers, so you may want to remove those you don't like.
to do that, select the first line below the object name and clic on the crosses at the right of the unwanted options, below.

**Reply by Bfinny** [reply]
Thank you for your help.
I didnt realize that you could change settings for the modifiers.
